src/custom/pages/About/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import useMarkdown from '../../hooks/useMarkdown'
+import { Title, Content, AppBodyMod } from './About.styled'
+import markdownContent from './about.md'
+import ReactMarkdown from 'react-markdown'
+
+export default function About() {
+  return (
+    <AppBodyMod>
+      <Title>About</Title>
+      <Content>
+        <ReactMarkdown>{useMarkdown(markdownContent)}</ReactMarkdown>
```

Comment:
Use <ReactMarkdown> here, instead of getting it from the custom hook (useMarkdown) to keep the custom hook as much as possible reusable?